web-common/src/features/custom-dashboards/VegaLiteRenderer.svelte

```diff
@@ -0,0 +1,33 @@
+<script lang="ts">
```

Comment:
Can we reuse the render in features/charts ? If it is significantly different, better to rename this component with `Embed` in it.